### PR TITLE
Update Hostgroup and Docker/Containers test metadata

### PR DESCRIPTION
**Pull request by mirzal**
Update Hostgroup and Docker/Containers test metadata. `testimony validate` clean; `testimony tiers` complains a bit, but only in tests that are part of Container Management - feature that is removed in Satellite 6.6. These tests will be removed before 6.6 GA.

Also cleaned up decorators a bit (removed `skip_if_bug_open` for bugs fixed long time ago), and changed order of imports where local linter complained.

```
$ for i in tests/foreman/api/test_docker.py tests/foreman/cli/test_docker.py tests/foreman/cli/test_repository.py tests/foreman/ui_airgun/test_container.py tests/foreman/ui_airgun/test_containerimagetag.py tests/foreman/api/test_hostgroup.py tests/foreman/cli/test_hostgroup.py tests/foreman/ui_airgun/test_hostgroup.py ; do testimony -c testimony.yaml validate "$i" ; done
Total number of tests: 51
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
Total number of tests: 62
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
Total number of tests: 94
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
Total number of tests: 3
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
Total number of tests: 1
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
Total number of tests: 46
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
Total number of tests: 36
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
Total number of tests: 3
Total number of invalid docstrings: 0 (00.00%)
Test cases with no docstrings: 0 (0.00%)
Test cases missing minimal docstrings: 0 (0.00%)
Test cases with unexpected tags: 0 (0.00%)
Test cases with unexpected token values in docstrings: 0 (0.00%)
Test cases with unparseable docstrings: 0 (0.00%)
```

### CODE REVIEW

File: tests/foreman/api/test_hostgroup.py [hunk 1]
```diff
@@ -5,26 +5,25 @@
...
 :TestType: Functional
 
 :CaseImportance: High
 
 :Upstream: No
 """
-from fauxfactory import gen_string
-from nailgun import client, entities, entity_fields
 from random import randint
+from fauxfactory import gen_string
```
Comment (vijay8451) [inline tests/foreman/api/test_hostgroup.py]: Please set this one in alphabetic order

File: tests/foreman/api/test_hostgroup.py [hunk 2]
```diff
@@ -5,26 +5,25 @@
...
 from requests.exceptions import HTTPError
+from nailgun import client, entities, entity_fields
```
Comment (vijay8451) [inline tests/foreman/api/test_hostgroup.py]: same as above

File: tests/foreman/api/test_hostgroup.py [hunk 3]
```diff
@@ -389,7 +385,7 @@ def test_positive_create_with_ptable(self):
         ).create()
         self.assertEqual(hostgroup.ptable.read().name, ptable.name)
 
-    @tier1
+    @tier2
```
Comment (vijay8451) [inline tests/foreman/api/test_hostgroup.py]: can we not keep as `tier1` as it's very basic functionality of component ? what do you think ?

same for below changes where it changed ..

File: tests/foreman/api/test_hostgroup.py (hunk 1 above)
Comment (mirzal) [reply tests/foreman/api/test_hostgroup.py]: According to PEP8, import order should be:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

So, `random` should be on top. But you are right, `nailgun` should be above `requests`. This is fixed now, and consistent with order with couple of other files I have checked (like `test_computeresource_libvirt.py`).

Comment (vijay8451) [reply tests/foreman/api/test_hostgroup.py]: yeah that's true and overall idea was to set in alphabetic order (as you mentioned above order only)

File: tests/foreman/ui_airgun/test_container.py [hunk 4]
```diff
@@ -4,13 +4,11 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
-
```
Comment (omkarkhatavkar) [inline tests/foreman/ui_airgun/test_container.py]: `CaseLevel: Acceptance ` removing from module level? We don't want to keep some default?

Comment (mirzal) [reply tests/foreman/ui_airgun/test_container.py]: That value is overwritten for every single test in this file.

Then, these tests are covering feature that is removed in 6.6, so they are going to be removed anyway. Probably whole file will be deleted.

And finally, if anyone ever adds new test there and forgets to set CaseLevel, `testimony` will complain about missing required token. So they will know to set CaseLevel for their test, or set default for entire file.

If these arguments don't convince you, let me know, I will add this line back.

Comment (omkarkhatavkar) [reply tests/foreman/ui_airgun/test_container.py]: Hmm, make sense! leave it then

**Pull request merged**